tools/dependency_summary.py

```diff
@@ -210,6 +210,13 @@
             "fixup": "https://raw.githubusercontent.com/remram44/adler32-rs/master/LICENSE",
         }
     },
+    "bincode": {
```

Comment:
CI is failing for `bincode`. I could not figure out why. So I pinned the license.